src/components/Blog/MediumPostPreview.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react'
+import styled from 'styled-components'
+import { format } from 'date-fns'
+import breakpoint from 'styled-components-breakpoint'
+
+import { Row, Col } from '../grid'
+import MediumLogo from '../../images/medium-logo'
+import { CardTitle, BodyPrimary } from '../Typography'
+import StyledLink from '../Common/StyledLink'
+import RatioContainer from '../Common/RatioContainer'
+import Image from '../Common/Image'
+import Anchor from '../Common/Anchor'
+
+const AuthorMediumLink = styled(Anchor)`
+  color: ${({ theme }) => theme.colors.textLight};
+  text-decoration: underline;
+`
+
+const AuthorAndDate = styled.p`
+  color: ${({ theme }) => theme.colors.textLight};
+`
+
+const StyledMediumIcon = styled(MediumLogo)`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 100px;
+  height: 100px;
+
+  ${breakpoint('smallTablet')`
+      width: 80px;
+      height: 80px;
+  `}
+
+  ${breakpoint('tablet')`
+    width: 100px;
+    height: 100px;
+  `}
+`
+
+const MediumPostImage = styled(Image)`
+  position: absolute;
+  max-width: unset;
+  width: auto;
+  height: 100%;
+`
+
+const MediumRow = styled(Row)`
+  padding-top: ${({ theme }) => theme.space[4]};
+  padding-bottom: ${({ theme }) => theme.space[3]};
+  ${breakpoint('smallTablet')`
+    padding-bottom: ${({ theme }) => theme.space[4]};
+  `}
+`
+
+const ImageWrapper = styled.div`
+  position: relative;
+  padding-top: ${({ theme }) => theme.space[2]};
+  padding-bottom: ${({ theme }) => theme.space[2]};
+  ${breakpoint('smallTablet')`
+    padding-bottom: 0;
+    padding-top: 0;
+  `};
+`
+
+const StyledBodyPrimary = styled(BodyPrimary)`
+  padding-bottom: ${({ theme }) => theme.space[1]};
+  ${breakpoint('smallTablet')`
+    padding-bottom: 0;
+  `};
+`
+
+const TitleAndAuthorWrapper = styled.div`
+  display: none;
+
+  ${({ show, hide }) =>
+    breakpoint(show, hide)`
+      display: block;
+    `}
+`
+
+const TitleAndAuthor = ({
+  title,
+  author,
+  formattedDate,
+  postUrl,
+  authorUrl,
+  show,
+  hide
+}) => {
+  return (
+    <TitleAndAuthorWrapper show={show} hide={hide}>
+      <Anchor href={postUrl}>
+        <CardTitle>{title}</CardTitle>
+      </Anchor>
+      <AuthorAndDate>
+        {author && (
+          <AuthorMediumLink href={authorUrl}>{author.name}</AuthorMediumLink>
+        )}
+        {' • '}
+        {formattedDate}
+      </AuthorAndDate>
+    </TitleAndAuthorWrapper>
+  )
+}
+
+const MediumPostPreview = ({
+  title,
+  createdAt,
+  uniqueSlug,
+  author,
+  virtuals
+}) => {
+  const previewText =
+    virtuals.subtitle.length < 145
+      ? virtuals.subtitle
+      : virtuals.subtitle.slice(0, 145) + '...'
+  const formattedDate = format(createdAt, 'MMMM DD')
+
+  const imageUrl = `https://cdn-images-1.medium.com/max/2000/${
```

Comment:
<img width="745" alt="Screenshot 2019-07-30 at 13 20 41" src="https://user-images.githubusercontent.com/7108120/62102828-f7aea080-b2cc-11e9-892a-f5bf385eabca.png">
`/max/2000/` in the URL will request all images at 2000px width which as you can see from the screenshot means they're all > 100kb and will slow down rendering the page for mobile users on slow connections. Because we're not using gatsby-image here there is no sort of 

From a bit of checking I think that the max width these images can go to is 480px so can you replace the url with something like 500px. I appreciate that the images come at different aspect ratios so you might need to do a bit checking to make sure they all take up the 1:1 container we've made, but I'm fairly confident they will!